nipype/algorithms/confounds.py

```diff
@@ -1186,35 +1267,72 @@ def compute_noise_components(imgseries, mask_images, num_components,
                 voxel_timecourses, repetition_time, period_cut)
         elif filter_type in ('polynomial', False):
             # from paper:
-            # "The constant and linear trends of the columns in the matrix M were
-            # removed [prior to ...]"
+            # "The constant and linear trends of the columns in the matrix M
```

Comment:
Please roll back these stylistic changes, they divert the attention during review.